docs/diagrams/PaymentTrackingActivityDiagram.puml

```diff
@@ -0,0 +1,27 @@
+@startuml
+skin rose
+skinparam ActivityFontSize 15
+skinparam ArrowFontSize 12
+start
+:User conducted a lesson;
```

Comment:
Maybe don't need this part. Can just go from the start to the diamond straight

Reply:
Might be because he want to show this part? Where student pays for today's lesson, or pay for owe amount
![image](https://github.com/user-attachments/assets/1ed99f47-5530-4f93-bd5d-8a90177bfcec)

Reply:
Ok that makes sense. Sorry I missed that.